lib/daru/io/importers/sql.rb

```diff
@@ -68,23 +68,23 @@ def call(query)
 
         private
 
-        def attempt_sqlite3_connection(db)
```

Comment:
`db` is totally good name. Please add this to `Naming/UncommunicativeMethodParamName:` cop's `AllowedNames:` settings.

In general, if Rubocop is starting to fail on code unrelated to your PR, it is better to notify maintainer (@athityakumar) than judge yourself.